src/odemis/acq/align/turnOnLight.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms  of the GNU General Public License version 2 as published by the Free
+Software  Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY;  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR  PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+
+from concurrent.futures._base import CancelledError, CANCELLED, FINISHED, RUNNING
+from odemis.model import CancellableFuture, logging
+import numpy
+from odemis.util import executeAsyncTask, img
+import threading
+
+
+def turnOnLightAndCheck(bl, ccd):
```

Comment:
Please update the test case to take into account this move. I'd suggest to even move the test case to its own "light_test.py" file.